setup.py

```diff
@@ -76,7 +76,6 @@
             "discussion = lms.djangoapps.discussion.apps:DiscussionConfig",
             "grades = lms.djangoapps.grades.apps:GradesConfig",
             "plugins = openedx.core.djangoapps.plugins.apps:PluginsConfig",
-            "schedules = openedx.core.djangoapps.schedules.apps:SchedulesConfig",
```

Comment:
Also remove from the `cms.djangoapp` listing?

Reply:
good call